book/nushell_map_functional.md

```diff
@@ -4,44 +4,45 @@ The idea behind this table is to help you understand how Nu builtins and plugins
-| skip while   | drop-while                   | dropWhile                       | dropWhile, dropWhileEnd  |
-| sort-by      | sort, sort-by, sorted-set-by | sort, sortBy, sortWith          | sort, sortBy             |
-| split row    | split, split-{at,with,lines} | split, words, lines             | split, words, lines      |
-| str          | clojure.string functions     | String functions                |                          |
-| str join     | join                         | concat                          | intercalate              |
-| str trim     | trim, triml, trimr           | trim, trimLeft, trimRight       | strip                    |
-| sum          | apply +                      | sum                             | sum                      |
-| take         | take, drop-last, pop         | take, init                      | take, init               |
-| take until   | take-while                   | takeWhile                       | takeWhile                |
-| take while   | take-while                   | takeWhile                       | takeWhile                |
-| uniq         | set                          | Set.empty                       | Data.Set                 |
-| where        | filter, filterv, select      | filter, filterMap               | filter                   |
+| Nushell      | Clojure                      | Tablecloth (Ocaml / Elm)        | Haskell                  | Nix                            |
+| ------------ | ---------------------------- | ------------------------------- | ------------------------ | ------------------------------ |
+| append       | conj, into, concat           | append, (++), concat, concatMap | (++)                     |  `builtins.concatLists`        |
+| into binary  | Integer/toHexString          |                                 | showHex                  |  `toString (toHex value)`      |
+| count        | count                        | length, size                    | length, size             |  `builtins.length`             |
+| date         | java.time.LocalDate/now      |                                 |                          |  `builtins.time`               |
+| each         | map, mapv, iterate           | map, forEach                    | map, mapM                |  `map`                         |
+| exit         | System/exit                  |                                 |                          |  `throw`                       |
+| first        | first                        | head                            | head                     |  `builtins.head`               |
+| format       | format                       |                                 | Text.Printf.printf       |  `builtins.toString`           |
+| group-by     | group-by                     |                                 | group, groupBy           |  Custom function with `map`    |
```

Comment:
This table should help folks quickly find the relevant Nushell command or construct if they have a particular language construct or idiom from other languages in mind or understand the meaning at a glance, so it should probably be left blank if you would have to come up with something custom in the other language. Its docs for Nushell not for Nix.